packages/calcite-components/src/components/date-picker-day/date-picker-day.scss

```diff
@@ -1,5 +1,6 @@
 :host {
```

Comment:
Add JSDocs for external CSS vars (tokens).

Reply:
Since the date-picker-day is internal. component, we don't need to document them.

Reply:
Updated: added JSDoc for CSS vars as per discussion in teams.